server/services/GithubWorkflowService.js

```diff
@@ -118,6 +118,32 @@ const fetchInstallationAccessToken = async (jsonWebToken, installationID) => {
   return JSON.parse(response.data).token;
 };
 
+/**
+ * We just want the whole number of the macOS version
+ * due to limitations on Github workflow runners
+ * See https://github.com/w3c/aria-at-app/issues/1143 for more info
+ *
+ * @param {object} atVersion
+ * @param {string} atVersion.name
+ */
+const getMacOSVersionName = atVersion => {
+  return atVersion?.name?.split('.')[0];
+};
+
+/**
+ * @param {string} atKey
+ * @param {object} atVersion
+ * @param {string} atVersion.name
+ */
+const uniqueWorkflowByAtVersion = (atKey, atVersion) => {
+  if (atKey === 'voiceover_macos') {
```

Comment:
Just to simplify the logic a bit here, we probably don't need to handle any other `atKeys` at the moment... but I can see how this might be useful in the future!

Reply:
Yep that was the thought but I can simplify!